common/data/content-format-ids.ts

```diff
@@ -1,11 +1,13 @@
 export const ArticleFormatIds = {
-  ImageGallery: 'W5uKaCQAACkA3C0T',
-  // Essay seems to signify an article that is part of a series
-  Essay: 'W7TfJRAAAJ1D0eLK',
+  ImageGallery: 'W5uKaCQAACkA3C0T', // Should this be called In Pictures?  https://wellcomecollection.prismic.io/documents~k=article-formats&b=working&c=published&l=en-gb/W5uKaCQAACkA3C0T/
```

Comment:
@gestchild @davidpmccormick didn't want to make that call right away, it doesn't _seem_ to be used but just double checking with you? Also added missing ones below.

Reply:
We should probably change it so it is consistent with Prismic to avoid confusion

Reply:
Turns out it was referenced once so changed it there too https://github.com/wellcomecollection/wellcomecollection.org/pull/9354/commits/cd48b31188f27d70ba372f158a2d447c1499cc97